electrum/gui/qt/transaction_dialog.py

```diff
@@ -245,6 +251,9 @@ def insert_tx_io(
                 tcf_shortid=tcf_shortid,
                 short_id=str(txin.short_id), addr=addr, value=txin_value,
             )
+            if isinstance(txin, PartialTxInput):
+                if txin.sighash != Sighash.ALL and self.wallet.adb.is_mine(addr):
+                    self.sighash_label.setText(_('Warning: Non-standard Sighash!'))
```

Comment:
Users won't know what this means. If we don't explain it (e.g. using a mouse-over tooltip), then we should use stronger wording at least. For example, instead of non-standard, we could say "dangerous".

Reply:
Sure, the text can be changed, but I think it also needs an extra confirmation step, e.g. add a confirmation pop-up before signing. Maybe even one extra safeguard for dangerous sighash types like SIGHASH_NONE, which can only be confirmed if first enabling an option in the preferences for instance.